sass/app.scss

```diff
@@ -1,5 +1,16 @@
 @import 'bower_components/reset-css/_reset';
 
+@mixin box-shadow($x, $y, $blur, $color) {
```

Comment:
box-shadow, have 5 parameters.
---------------------
Horizontal Length
Vertical Length
Blur Radius
Spread Radius

Please add them all, and make last Spread Radius - as unnecessary